go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -93,7 +91,13 @@ func NewController(client clientset.Interface,
 	c.eventRecorder = recorder
 
 	// repair controller
-	c.repair = NewRepair(0, serviceInformer.Lister(), clusterPortGroupUUID)
+	c.repair = newRepair(serviceInformer.Lister())
+
+	// load balancers need to be applied to nodes, so
+	// we need to watch Node objects for changes.
+	c.nodeTracker = newNodeTracker(nodeInformer)
+	c.nodeTracker.resyncFn = c.RequestFullSync
```

Comment:
I think a note that RequestFullSync is only called on a switch GR event would be helpful here